src/main/java/frc/robot/RobotContainer.java

```diff
@@ -267,6 +267,11 @@ public void updateUserInput() {
     // }
     else if (input.controllerButton(7)) {
       superstructure.resetRobot();
+    } else if (input.rightJoystickButton(15)) {
+      DriveCommands.SnapToTarget(
+              drive, () -> -input.leftJoystickY(), () -> -input.leftJoystickX(), Targets.FLING)
+          .until(() -> input.rightJoystickButton(2));
```

Comment:
the until should be the same button as the button the driver is pressing (aka it should be `.until(() -> input.rightJoystickButton(15));` )

Reply:
Both of the issues should be fixed in the latest commit on fling